frontend/src/components/form/base/EColorPicker.vue

```diff
@@ -16,15 +16,30 @@ Displays a field as a color picker (can be used with v-model)
     @input="$emit('input', $event)"
   >
     <template #default="picker">
-      <v-card>
+      <v-card :style="{ '--picker-contrast-color': contrast }" data-testid="colorpicker">
         <v-color-picker
           :value="removeAlpha(picker.value || '')"
           flat
           @input="picker.onInput"
         />
-        <v-btn text color="primary" @click="picker.close">
```

Comment:
how is this in accessibility terms if there is no button to close the dialog?

Reply:
They can click outside or press <kbd>Esc</kbd>.